scapy/arch/linux.py

```diff
@@ -132,7 +133,8 @@ def _get_if_list():
     """
     try:
         with open("/proc/net/dev", "r") as f:
-            return [line.split(':', 1)[0].strip() for line in f.readlines()[2:]]
+            return [line.split(':', 1)[0].strip()
+                    for line in itertools.islice(f, 2, None)]
```

Comment:
What is the benefit of using `itertools.islice` vs. using a regular slice?

Reply:
Ah, never mind, I checked the help myself: the main difference is that the lazy evaluation.